pallets/identity-management/src/lib.rs

```diff
@@ -93,22 +93,22 @@ pub mod pallet {
 		// TODO: what if the event is triggered by an extrinsic that is included in a batch call?
 		//       Can we retrieve that extrinsic hash in F/E?
 		IdentityLinked {
-			account: T::AccountId,
+			identity: Identity,
```

Comment:
This might be misleading - it doesn't hold information about the identity that was linked, but idgraph's root identity.
What do you think about renaming it ? `primary_identity` or `idgraph_root_identity`  ?

Reply:
Good catch, thanks - I'll rename it (and other occurrences) to `prime_identity`